py-polars/polars/functions/lit.py

```diff
@@ -173,25 +182,21 @@ def lit(
     if dtype:
         return wrap_expr(plr.lit(value, allow_object, is_scalar=True)).cast(dtype)
 
-    try:
-        # numpy literals like np.float32(0) have item/dtype
-        item = value.item()
-
-        # numpy item() is py-native datetime/timedelta when units < 'ns'
-        if isinstance(item, (datetime, timedelta)):
+    if isinstance(value, np.generic):
```

Comment:
This made numpy a required package.

Reply:
Fixed: https://github.com/pola-rs/polars/pull/22486 ✌️